src/main/java/bussiness_layer/services/StudentService.java

```diff
@@ -0,0 +1,32 @@
+package bussiness_layer.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import bussiness_layer.dto.EnrollmentDto;
+import bussiness_layer.mappers.EnrollmentMapper;
+import data_layer.domain.Enrollment;
+import data_layer.repositories.IEnrollmentRepository;
+import utils.exceptions.ResourceNotFoundException;
+
+@Service
+@Transactional
+public class StudentService implements IStudentService {
+
+    @Autowired
+    private IEnrollmentRepository enrollmentRepository;
+
+
+    @Override
+    public List<EnrollmentDto> getEnrollments(String studUsername) {
```

Comment:
Write at least two test methods for this function, one for the happy-flow and one for checking if that exception is thrown. Create a StudentIT class in the test ->business_layer->integration